akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -480,8 +480,9 @@ object Partition {
    *
    * @param outputPorts number of output ports
    * @param partitioner function deciding which output each element will be targeted
+   * @param eagerCancel if true, partition cancels upstream if any of its downstreams cancel
    */
-  def apply[T](outputPorts: Int, partitioner: T ⇒ Int): Partition[T] = new Partition(outputPorts, partitioner)
+  def apply[T](outputPorts: Int, partitioner: T ⇒ Int, eagerCancel: Boolean = false): Partition[T] = new Partition(outputPorts, partitioner, eagerCancel)
```

Comment:
In general default params are a real pain in public APIs as they making evolving without breaking bincomp hard, please provide overloads instead.